pkg/models/org_user.go

```diff
@@ -119,6 +119,8 @@ type GetOrgUsersQuery struct {
 	OrgId  int64
 	Query  string
 	Limit  int
+	// Flag used to allow oss edition to query users without access control
+	DontEnforceAccessControl bool
```

Comment:
We cant inject licensing service into sqlstore due to circular dependency in enterprise. This flag is used as a hack and will only be true when running oss or enterprise without license for the /api/users/lookup path. If a user is able to set permissions for either a team, dashboard or folder they should also be able to look up users in org

Reply:
Didn't we decide that you still need additional permissions to look up users in order to grant them team/dashboard/folder permissions?

Reply:
Ok, as discussed offline - the check will still be carried out for Enterprise, but when calling the user listing endpoints from OSS it won't be used.

What about `GET /api/orgs/:orgId/users` and `GET /api/org/users` endpoints? Should those also set `DontEnforceAccessControl` to true if running without a license?

Reply:
Yeah maybe, but if we decide to allow them to i think we should do it in another pr, this endpoint is the only one covered by reqOrgAdminFolderAdminOrTeamAdmin when not using fgac

Reply:
To answer it - the other endpoints are only available to Grafana server admins, who have the permissions to list all the users. So it's ok to apply the auth filter for them.